integration-cli/docker_cli_by_digest_test.go

```diff
@@ -561,8 +561,12 @@ func (s *DockerRegistrySuite) TestPullFailsWithAlteredManifest(c *testing.T) {
 	out, exitStatus, _ := dockerCmdWithError("pull", imageReference)
 	assert.Assert(c, exitStatus != 0)
 
-	expectedErrorMsg := fmt.Sprintf("manifest verification failed for digest %s", manifestDigest)
-	assert.Assert(c, is.Contains(out, expectedErrorMsg))
+	if testEnv.UsingSnapshotter() {
+		assert.Assert(c, is.Contains(out, "unexpected commit digest"))
+		assert.Assert(c, is.Contains(out, "expected "+manifestDigest))
+	} else {
+		assert.Assert(c, is.Contains(out, fmt.Sprintf("manifest verification failed for digest %s", manifestDigest)))
```

Comment:
Clearly not for this PR, but I know that `manifest verification failed for digest` has been a good error to search for; perhaps we can wrap containerd's error to add that as prefix in the error we return 🤔